examples/protocols/displayport/displayport-main.stanza

```diff
@@ -11,6 +11,10 @@ defpackage jsl/examples/protocols/displayport/main:
 
   import jsl/protocols/displayport
   import jsl/si/helpers
+  import jsl/bundles
+  import jsl/si/constraints
+  import jsl/si/pairs
+  import jsl/si/couplers
```

Comment:
Why would we not use just `import jsl/si` here?

Reply:
Oversight